packages/server/database/migrations/20200303123942-actionToCheckIn.ts

```diff
@@ -0,0 +1,29 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function(r: R) {
```

Comment:
@mattkrick hey, the migration works, yay! Anything here look suspect? This is probably the first migration I’ve written.